nixnet/_session/base.py

```diff
@@ -335,6 +335,37 @@ def disconnect_terminals(self, source, destination):
         """
         _funcs.nx_disconnect_terminals(self._handle, source, destination)
 
+    def write_state_lin_schedule_change(self, state_value):
```

Comment:
`change_lin_schedule` makes sense and still conveys the essence of the function (at least from my understanding of it). @lilesj any thoughts?